diff --git a/adapter/parser.go b/adapter/parser.go
index 290a4a8f..06274154 100644
--- a/adapter/parser.go
+++ b/adapter/parser.go
@@ -33,27 +33,8 @@ func ParseProxy(mapping map[string]any, options ...ProxyOption) (C.Proxy, error)
 			break
 		}
 		proxy, err = outbound.NewShadowSocks(*ssOption)
-	case "ssr":
-		ssrOption := &outbound.ShadowSocksROption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, ssrOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewShadowSocksR(*ssrOption)
-	case "socks5":
-		socksOption := &outbound.Socks5Option{BasicOption: basicOption}
-		err = decoder.Decode(mapping, socksOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewSocks5(*socksOption)
-	case "http":
-		httpOption := &outbound.HttpOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, httpOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewHttp(*httpOption)
+
+
 	case "vmess":
 		vmessOption := &outbound.VmessOption{BasicOption: basicOption}
 		err = decoder.Decode(mapping, vmessOption)
@@ -68,13 +49,7 @@ func ParseProxy(mapping map[string]any, options ...ProxyOption) (C.Proxy, error)
 			break
 		}
 		proxy, err = outbound.NewVless(*vlessOption)
-	case "snell":
-		snellOption := &outbound.SnellOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, snellOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewSnell(*snellOption)
+
 	case "trojan":
 		trojanOption := &outbound.TrojanOption{BasicOption: basicOption}
 		err = decoder.Decode(mapping, trojanOption)
@@ -82,13 +57,7 @@ func ParseProxy(mapping map[string]any, options ...ProxyOption) (C.Proxy, error)
 			break
 		}
 		proxy, err = outbound.NewTrojan(*trojanOption)
-	case "hysteria":
-		hyOption := &outbound.HysteriaOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, hyOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewHysteria(*hyOption)
+
 	case "hysteria2":
 		hyOption := &outbound.Hysteria2Option{BasicOption: basicOption}
 		err = decoder.Decode(mapping, hyOption)
@@ -96,20 +65,7 @@ func ParseProxy(mapping map[string]any, options ...ProxyOption) (C.Proxy, error)
 			break
 		}
 		proxy, err = outbound.NewHysteria2(*hyOption)
-	case "wireguard":
-		wgOption := &outbound.WireGuardOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, wgOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewWireGuard(*wgOption)
-	case "tuic":
-		tuicOption := &outbound.TuicOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, tuicOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewTuic(*tuicOption)
+
 	case "direct":
 		directOption := &outbound.DirectOption{BasicOption: basicOption}
 		err = decoder.Decode(mapping, directOption)
@@ -131,41 +87,8 @@ func ParseProxy(mapping map[string]any, options ...ProxyOption) (C.Proxy, error)
 			break
 		}
 		proxy = outbound.NewRejectWithOption(*rejectOption)
-	case "ssh":
-		sshOption := &outbound.SshOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, sshOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewSsh(*sshOption)
-	case "mieru":
-		mieruOption := &outbound.MieruOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, mieruOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewMieru(*mieruOption)
-	case "anytls":
-		anytlsOption := &outbound.AnyTLSOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, anytlsOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewAnyTLS(*anytlsOption)
-	case "sudoku":
-		sudokuOption := &outbound.SudokuOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, sudokuOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewSudoku(*sudokuOption)
-	case "masque":
-		masqueOption := &outbound.MasqueOption{BasicOption: basicOption}
-		err = decoder.Decode(mapping, masqueOption)
-		if err != nil {
-			break
-		}
-		proxy, err = outbound.NewMasque(*masqueOption)
+
+
 	default:
 		return nil, fmt.Errorf("unsupport proxy type: %s", proxyType)
 	}
